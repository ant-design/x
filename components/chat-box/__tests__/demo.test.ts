import demoTest from '../../../tests/shared/demoTest';

demoTest('chat-box');
