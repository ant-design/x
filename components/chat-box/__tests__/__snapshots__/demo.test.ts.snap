// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders components/chat-box/demo/basic.tsx correctly 1`] = `
<div>
  2333
</div>
`;
